### ISSUE TITLE
{{since build.started}} does not reflect the actual build time in a pipeline

### ISSUE THREAD

**Issue by brlin-tw** [appleboy/drone-telegram#108]
The sent message indicates that the build time is 5s:

![image](https://user-images.githubusercontent.com/13408130/137115492-1e352eab-e309-4c7b-a0a3-2e3f91089310.png)

However, the actual pipeline takes 2m3s:

![image](https://user-images.githubusercontent.com/13408130/137116343-9d9e59ce-6767-4c42-9492-e41f0bc4d59f.png)

Step definition:

```yaml
  - name: 發送持續整合結果到告警頻道
    when:
      status:
        - success
        - failure
    depends_on:
      - 靜態代碼分析
    image: appleboy/drone-telegram
    settings:
      token:
        from_secret: telegram_bot_auth_token
      to: __REDACTED__
      format: markdown
      message: "**{{#success build.status}}✅ 持續整合(CI)通過**{{else}}❌ 持續整合(CI)不通過，請盡快修正問題{{/success}}**\n\n📖專案：{{repo.namespace}}/{{repo.name}}\n🎋分支：{{commit.branch}}\n👤作者：{{commit.author}}\n✒️修訂版：{{truncate commit.sha 8}}\n\n```\n{{commit.message}}\n```\n⏲️花費時間：{{since build.started}}\n🌐報告連結：{{build.link}}"
```

**Comment by sardaukar** [comment]
It seems to be the time for the telegram step only

**Comment by appleboy** [comment]
fixed in #114
